scripts/_common.sh

```diff
@@ -95,7 +95,8 @@ init_composer() {
     || ynh_die "Unable to install Composer"
 
   # update dependencies to create composer.lock
-  exec_composer "$DESTDIR" install  --no-dev --prefer-dist --no-scripts  \
+  exec_composer "$DESTDIR" update symfony/flex --no-plugins --no-scripts
+  exec_composer "$DESTDIR" install  --no-dev --prefer-dist --no-scripts \
```

Comment:
Why do you use `--no-scripts`? Symfony flex creates config files while installing the bundles. This might even break Symfony apps, as they rely on the scripts running.
The recommended way is to run `composer install --no-dev --optimize-autoloader`.

Reply:
I have changed composer install as you suggested to "install  --no-dev --optimize-autoloader"
Please see https://github.com/YunoHost-Apps/kimai2_ynh/pull/3/commits/436fe9d924e295bfdd262c3146585b58104d7743#diff-dc2801cb46a5d60e62234cff98084498R98 

and

https://github.com/YunoHost-Apps/kimai2_ynh/pull/3/commits/436fe9d924e295bfdd262c3146585b58104d7743#diff-dc2801cb46a5d60e62234cff98084498R120